.gitmodules

```diff
@@ -4,12 +4,12 @@
   branch = dev/emc
 [submodule "ccpp/framework"]
   path = ccpp/framework
-  url = https://github.com/NCAR/ccpp-framework
-  branch = main
+  url = https://github.com/uturuncoglu/ccpp-framework
```

Comment:
This needs to be:
```suggestion
  url = https://github.com/climbfuji/ccpp-framework
```

Reply:
@climbfuji thanks for pointing it. I forgot to update the submodule after testing your branch. I should be fixed now.

Reply:
@climbfuji done.